Jenkinsfile

```diff
@@ -44,17 +44,40 @@ pipeline {
       }
     }
 
-    stage('Test Docker image') {
-      steps {
-        sh './test.sh'
-        sh 'cd ci && ./test-ldap'
+    stage('Run Tests') {
+      parallel {
+        stage('RSpec') {
+          steps { sh 'cd ci && ./test --rspec' }
+        }
+        stage('Authenticators') {
+          steps { sh 'cd ci && ./test --cucumber-authenticators' }
+        }
+        stage('Policy') {
+          steps { sh 'cd ci && ./test --cucumber-policy' }
+        }
+        stage('API') {
+          steps { sh './test.sh' }
```

Comment:
This could maybe use a comment (with TODO) indicating why it's different from the others.